components/webhook/EventTypes.tsx

```diff
@@ -11,9 +11,11 @@ export const eventTypes = [
 const EventTypes = ({
   onChange,
   values,
+  error,
 }: {
   onChange: React.ChangeEventHandler<HTMLInputElement>;
   values: WebookFormSchema['eventTypes'];
+  error: string | string[] | undefined;
```

Comment:
@jatingodnani Can it be ever `string[]` in this case?

Reply:
Error is a string only

Reply:
I think it is flexible to write this @devkiran

Reply:
@jatingodnani Not sure what do you mean? If it can be array, should we display the more than one errors, `<div className="text-[12px] text-red-500">{error}</div>` this won't work in that case. Please check it from your side.

Reply:
@devkiran  if i am using string |undefined it is working on localhost but at the build-time it showing error .you can check above commit